homeassistant/components/sensor/qwikswitch.py

```diff
@@ -0,0 +1,63 @@
+"""
+Support for Qwikswitch Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.qwikswitch/
+"""
+import logging
+
+from homeassistant.components.qwikswitch import DOMAIN as QWIKSWITCH
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = [QWIKSWITCH]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, _, add_devices, discovery_info=None):
+    """Add lights from the main Qwikswitch component."""
+    qsusb = hass.data[QWIKSWITCH]
+    _LOGGER.info("Setup qwikswitch.sensor %s, %s", qsusb, discovery_info)
+    devs = [QSSensor(name, id)
+            for name, id in discovery_info[QWIKSWITCH].items()]
```

Comment:
Is `id` a list of ids or just one id?

Reply:
Just a single id. The dict member can repeat the entity_id and qs_id {entity_id: qs_id} if the sensor class can understand it